core/dataloader.py

```diff
@@ -113,15 +111,22 @@ def __getitem__(self, idx, is_pickle=True):
         scene_path = self.all_files[scene_num]
         data = pickle.load(open(scene_path, "rb"))
         
-        viewpoints = torch.tensor(data['origin_T_camXs_raw'])
+        #gt egomotion
```

Comment:
#13

Reply:
Let's do a sanity check on it later. Merging the PR for now